EpiAware/docs/Project.toml

```diff
@@ -1,4 +1,16 @@
 [deps]
 Changelog = "5217a498-cd5d-4ec6-b8c2-9b85a09b6e3e"
+DataFramesMeta = "1313f7d8-7da2-5740-9ea0-a2ca25f37964"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-EpiAware = "b2eeebe4-5992-4301-9193-7ebc9f62c855"
```

Comment:
Could you explain again (and maybe point at the docs) why this is needed because it is horrible.

Reply:
First let me check if it is _actually_needed

Reply:
You can make this better by using `dev`; see the usage [here](https://pkgdocs.julialang.org/v1/managing-packages/#developing).

This is fine locally; just checked. But for CI we might want to run `dev` on `EpiAware` before making the documentation.

Essentially, engineering has been done that interacting with the `Test` environment is clever and understand that we need to load the _developing_  code of `EpiAware`. 

The `doc` environment engineering seems to come from a use-case of we've made a package, put it on Github and probably Julia registry, and _now_ we want nice docs.

Another possibility is to `add` `EpiAware` direct from this open repo; in CI this would always get the most recent version (I think).

Reply:
```julia
using Pkg
 Pkg.add(url="https://github.com/CDCgov/Rt-without-renewal/", subdir="EpiAware")
```

Reply:
I might try that pattern actually

Reply:
> You can make this better by using dev; see the usage 

So this is what I thought I had done but maybe not?

Reply:
> direct from this open repo; in CI this would always get the most recent version (I think).

We will have the code checked out in the action so we could add it not with a URL. that would be more stable/less likely to break